test/unit/strategies/extensions/MilestonesExtensionUnit.t.sol

```diff
@@ -0,0 +1,424 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.19;
+
+import {Test} from "forge-std/Test.sol";
+import {MockMockMilestonesExtension} from "test/smock/MockMockMilestonesExtension.sol";
+import {IMilestonesExtension} from "contracts/strategies/extensions/milestones/IMilestonesExtension.sol";
+import {Metadata} from "contracts/core/libraries/Metadata.sol";
+
+contract MilestonesExtensionUnit is Test {
+    MockMockMilestonesExtension milestonesExtension;
+
+    struct MilestoneWithoutEnums {
+        uint256 amountPercentage;
+        Metadata metadata;
+        uint8 status;
+    }
+
+    function _parseMilestones(MilestoneWithoutEnums[] memory _rawMilestones)
+        internal
+        view
+        returns (IMilestonesExtension.Milestone[] memory _milestones)
+    {
+        _milestones = new IMilestonesExtension.Milestone[](_rawMilestones.length);
+        for (uint256 i = 0; i < _milestones.length; i++) {
+            _milestones[i].amountPercentage = bound(_rawMilestones[i].amountPercentage, 1, type(uint128).max);
+            _milestones[i].metadata = _rawMilestones[i].metadata;
+            _milestones[i].status = IMilestonesExtension.MilestoneStatus(bound(uint256(_rawMilestones[i].status), 0, 6));
+        }
+    }
+
+    function setUp() public {
+        milestonesExtension = new MockMockMilestonesExtension(address(0));
+    }
+
+    function test___MilestonesExtension_initShouldCall_increaseMaxBid(uint256 _maxBid) external {
+        milestonesExtension.mock_call__increaseMaxBid(_maxBid);
+
+        // It should call _increaseMaxBid
+        milestonesExtension.expectCall__increaseMaxBid(_maxBid);
+
+        milestonesExtension.call___MilestonesExtension_init(_maxBid);
+    }
+
+    function test_IncreaseMaxBidWhenParametersAreValid(uint256 _maxBid) external {
+        milestonesExtension.mock_call__increaseMaxBid(_maxBid);
+        milestonesExtension.mock_call__checkOnlyPoolManager(address(this));
+
+        // It should call _checkOnlyPoolManager
+        milestonesExtension.expectCall__checkOnlyPoolManager(address(this));
+
+        // It should call _increaseMaxBid
+        milestonesExtension.expectCall__increaseMaxBid(_maxBid);
+
+        milestonesExtension.increaseMaxBid(_maxBid);
```

Comment:
Can we also call the getter and assert it is equal to the new max bid?

Reply:
I'm not sure if that is correct since we are testing that on `_increaseMaxBid()` [here](https://github.com/allo-protocol/allo-v2.1/blob/03c9507784196c0e6ea9302afa36802cc2c0539c/test/unit/strategies/extensions/MilestonesExtensionUnit.t.sol#L380)